cime_config/usermods_dirs/NEON/defaults/user_nl_clm

```diff
@@ -18,9 +18,6 @@
 ! Set glc_do_dynglacier  with GLC_TWO_WAY_COUPLING               env variable
 !----------------------------------------------------------------------------------
 
-flanduse_timeseries = ' '   ! This isn't needed for a non transient case, but will be once we start using transient compsets
-fsurdat = "$DIN_LOC_ROOT/lnd/clm2/surfdata_map/NEON/surfdata_1x1_NEON_${NEONSITE}_hist_78pfts_CMIP6_simyr2000_c230601.nc"
```

Comment:
I have now merged #2472 and #2435; however, believe it or not the very last test on izumi (that was pending and I decided to not wait for) shows a problem that may be due to the removal of fsurdat here...
```
ERROR : CLM build-namelist::CLMBuildNamelist::add_default() : No default value found for fsurdat.
            Are defaults provided for this resolution and land mask?'
```
@wwieder how would you like to address this?

Reply:
I should have mentioned the test:
`SMS_Ld10_D_Mmpi-serial.CLM_USRDAT.I1PtClm51Bgc.izumi_nag.clm-NEON-MOAB--clm-PRISM.C.0416-112218iz_nag`

Reply:
I'd suggest not removing the surface dataset if that gets the test to pass.  Is that simple enough to add back in?

Reply:
I will try it tomorrow. If that fixes it, I think simplest is for me to open a new PR with the one-line change and merge into b4b-dev.